packages/xo-server-perf-alert/src/RrdHostVm.js

```diff
@@ -104,7 +104,7 @@ export class RrdHostVm extends MonitorStrategy {
         return
       }
       const [, type, uuidInStat, metricType] = matches
-      if (!USED_METRICS.includes(metricType)) {
+      if (!USED_METRICS.includes(metricType) || metricType.match('/cpu[0-9]+')) {
```

Comment:
Unless I'm missing something, wouldn't the string` '/cpu[0-9]+' `look for a literal slash character at the beginning? Since `metricType` values coming from the RRD legend parser (e.g., cpu0, cpu1) don't contain a leading slash, wouldn't this condition never match?

 Should this be changed to a proper RegExp literal instead?
```suggestion
      if (!USED_METRICS.includes(metricType) || metricType.match(/cpu[0-9]+/)) {
```